### PR TITLE
Issue/bug23

### PR DESCRIPTION
**Pull request by Logan-K**
Rows look better!

### CODE REVIEW

File: app/views/planned_runs/index.html.erb [hunk 1]
```diff
@@ -26,21 +23,21 @@
...
           <div class="row" style="background-color: lightblue">
       <% end %>
-        <div class="col-md-3 col-sm-3 col-xs-3">
+        <div class="col-md-4 col-sm-3 hidden-xs index_rows">
           <%= planned_run.date %>
         </div>
-        <div class="col-md-3 col-sm-3 col-xs-2">
-          <%= Group.find(planned_run.group_id).name %>
+        <div class="hidden-lg hidden-md hidden-sm col-xs-3 index_rows">
+          <%= Date.strptime(planned_run.date.to_s, "%Y-%m-%d").strftime("%m-%d") %>
```
Comment (epugh) [inline app/views/planned_runs/index.html.erb]: This sholud be a Helper I think...   Embedding Ruby code it's kind of icky.

File: app/views/runners/index.html.erb [hunk 2]
```diff
@@ -16,30 +16,30 @@
...
-  <div class="col-md-4 col-sm-4 col-xs-5">
+  <div class="col-md-4 col-sm-4 col-xs-4">
     <strong>OPTIONS</strong>
   </div>
   <% end %>
 </div>
 
 <% @runners.each.with_index do |runner, index| %>
-  <% if index % 2 ==0 %>
+  <% if index % 2 == 0 %>
```
Comment (epugh) [inline app/views/runners/index.html.erb:26]: Isn't there some sort of bootstrap way to do alternativing rows via CSS?

**Comment by epugh** [comment]
Checkout the local_time helper that we used on Horsified: https://github.com/o19s/horsified/blob/master/app/views/reporting/announcers_list.html.erb#L11

It made it very easy to store the times in UTC, but convert to local.  I think I used it as well on the activity feed...

File: app/views/planned_runs/index.html.erb (hunk 1 above)
Comment (Logan-K) [inline app/views/planned_runs/index.html.erb]: Just used .strftime and it seemed to do the job: https://github.com/o19s/thats-trackable/commit/7c652bd8b733f75dc6a5aac434191e759925dea5

File: app/views/runners/index.html.erb (hunk 2 above)
Comment (Logan-K) [inline app/views/runners/index.html.erb:26]: Hmm so there is a bootstrap "striped" table here: http://getbootstrap.com/css/#tables-striped but that would be in use with tables which I've decided to not use. It is possible to do with css but it starts getting into some intense css. Here is a fiddle of a potential way: http://jsfiddle.net/aniskhan001/32KEX/ I'd honestly like to know what you would do. Shall I "css-atize" it or should I keep it as logic on the view pages?

**Pull request merged**